src/views/Patient/Patient.tsx

```diff
@@ -0,0 +1,57 @@
+import './Patient.css';
+import { Box, Card, CardContent, Typography } from '@mui/material';
+import { useEffect, useState } from 'react';
+
+function Patient(props: any) {
+  const client = props.client;
+
+  const [patient, setPatient] = useState(null);
+
+  useEffect(() => {
+    client.patient.read().then((patient: any) => setPatient(patient));
+  });
+
+  // console.log(patient);
+
+  return (
+    <div className='Patient'>
+      <Box sx={{
+        marginTop: 8,
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+      }}>
+        {/* Demo of data from fhir server */}
+        <p>Launched with patient context:</p>
+        {patient ?
+          <Card sx={{ minWidth: 500, maxWidth: 5000, bgcolor: '#F6F6F6', p:5 }}>
+            <CardContent>
+              <Typography sx={{ fontSize: 14 }} color="text.secondary" gutterBottom>
+                Patient Info
+              </Typography>
+              <Typography variant="h5" component="div">
+                {patient['name'][0]['given'][0]} {patient['name'][0]['family']}
```

Comment:
Once the patient is typed you can access its properties directly and can use optional chaining to access these properties. 

`patient.name?.[0]?.given?.[0]` for example, to access the given name.  Then the same concept for the family name and address information.